### PR TITLE
Hide loader on start flow dialog when there are no channels

### PR DESCRIPTION
**Pull request by norkans7**
Fixes https://github.com/rapidpro/rapidpro/issues/1124

### CODE REVIEW

**Comment by norkans7** [comment]
Pick a Number 2020-02-11 17-54-36

File: templates/flows/flow_broadcast.haml [hunk 1]
```diff
@@ -56,6 +56,8 @@
       -trans "You can always test your flow using the simulator, click"
         %span.glyph.icon-mobile{style:"padding-top:2px;"}
       -trans "on the right to open it."
+    :javascript
+      $(".loader").hide();
```
Comment (rowanseymour) [inline templates/flows/flow_broadcast.haml]: don't we need this in the `elif` block below as well? 

I'm confused where is `.loader` defined...

Comment (norkans7) [reply templates/flows/flow_broadcast.haml]: It is from https://github.com/nyaruka/rapidpro/blob/master/templates/modal.haml#L17 and normally when we load a view that loader will be hidden once the view is fetched and displayed.

Correct we need that in the elif too

Comment (rowanseymour) [reply templates/flows/flow_broadcast.haml]: and why doesn't it get hidden in this particular case?

Comment (norkans7) [reply templates/flows/flow_broadcast.haml]: I'll investigate more on that

**Comment by norkans7** [comment]
Replaced by https://github.com/nyaruka/rapidpro/pull/2687

**Pull request closed without merging**